common/src/WooCommerce.php

```diff
@@ -132,19 +132,34 @@ private function sendInvite($order_id) {
         }
 
         // send invite
+        $is_invite_added = true;
         $api = new API($api_domain, $shop_id, $api_key);
         try {
             $api->invite($data);
-        } catch (WebwinkelKeurAPIAlreadySentError $e) {
-            // that's okay
-        } catch (WebwinkelKeurAPIError $e) {
-            $this->logApiError($e);
+        } catch (Exception $e) {
+            $is_invite_added = false;
+            if ($e instanceof WebwinkelKeurAPIAlreadySentError) {
```

Comment:
Currently `WebwinkelKeurAPIAlreadySentError` never gets thrown.
https://github.com/webwinkelkeur/wordpress/blob/3f90dec6b2c79a795cc9491309b0f6097322c6c0/common/src/API.php#L50
The response for `Invitation already sent for this order.` is `success`

Reply:
Is this intentional?

Reply:
Not really, it's probably that way because of the old pre-dashboard API, which didn't use JSON.  Those two checks on the message can probably be removed.

https://github.com/webwinkelkeur/wordpress/blob/3f90dec6b2c79a795cc9491309b0f6097322c6c0/common/src/API.php#L53-L58